src/styles/main.scss

```diff
@@ -1,3 +1,355 @@
+}
+
+.header__title {
+  margin: 0;
+  font-size: 36px;
+  font-family: Space Grotesk, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 100%;
+}
+
+.header__bottom {
+  padding: 0 20px 40px;
+}
+/* #endregion */
+/* #region menu */
+/* #region topbar */
+.topbar {
+  padding: 20px 20px 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.topbar__logo {
+  display: flex;
+}
+
+.topbar__icons {
+  display: flex;
+  gap: 24px;
+}
+/* # endregion */
+
+.menu {
+  box-sizing: border-box;
+  height: 100vh;
+  background-color: #e5e6e8;
+}
+
+.menu__top {
+  margin-bottom: 32px;
+}
+
+.menu__content {
+  padding-inline: 20px;
+}
+
+.menu__phone-number {
+  color: inherit;
+  font-size: 16px;
+  line-height: 140%;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+  margin-top: 48px;
+}
+
+.menu__call-to-order {
+  text-decoration: none;
+  color: inherit;
+  font-size: 16px;
+  line-height: 140%;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  display: block;
+  margin-top: 16px;
+  padding-bottom: 4px;
+  border-bottom: 1px solid #292929;
+  width: fit-content;
+}
+/* #endregion */
+/* #region nav */
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 22px;
+  letter-spacing: 2px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: inherit;
+}
+
+/* #endregion */
+/* #region icon */
+.icon {
+  display: block;
+  height: 24px;
+  width: 24px;
+}
+
+.icon--phone {
+  background-image: url(../icons/phone_icon.png);
+}
+
+.icon--menu {
+  background-image: url(../icons/menu_icon.png);
+}
+
+.icon--close {
+  background-image: url(../icons/close_icon.png);
+}
+/* #endregion */
+/* region main */
+.main {
+  padding: 0 20px;
+}
+/* endregion */
+/* #section region */
+.section__title {
+  color: #292929;
+  font-size: 32px;
+  font-family: Space Grotesk, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 32px;
+  margin: 0;
+  margin-bottom: 8px;
+}
+
+.section__subtitle {
   margin: 0;
+  margin-top: 24px;
+  color: #292929;
+  font-size: 18px;
+  font-family: Space Grotesk, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 140%;
+}
+
+.section__info {
+  margin: 0;
+  margin-top: 8px;
+  color: #292929;
+  font-size: 16px;
+  font-family: Space Mono, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 140%;
+}
+
+.products {
+  margin-top: 64px;
+}
+
+.products__item {
+  margin-top: 40px;
+}
+
+.products__link {
+  display: flex;
+}
+
+.products__link--img {
+  margin-top: 40px;
+}
+
+.products__price {
+  margin: 0;
+  margin-top: 12px;
+  color: #292929;
+  font-size: 16px;
+  font-family: Space Grotesk, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 140%;
+}
+
+/* #endregion */
+/* region store */
```

Comment:
The region comment for the main section is not correctly commented. 
It should be written as `/* #region main */` instead of `/* region main */`.